<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowMain</class>
 <widget class="QMainWindow" name="windowMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>326</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>NoLimitsStudio</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="GLGrid"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>519</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionImportNL"/>
    <addaction name="actionExport_NoLimits_Track"/>
    <addaction name="separator"/>
    <addaction name="actionBake_for_NoLimits"/>
    <addaction name="actionStart_Simulator"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionCAD_Mode"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Sky"/>
    <addaction name="actionShow_Ground"/>
    <addaction name="actionShow_Terrain"/>
    <addaction name="actionShow_Supports"/>
    <addaction name="actionShow_Track"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuView"/>
   <addaction name="menuDisplay"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImportNL">
   <property name="text">
    <string>import *.nltrack</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport_NoLimits_Track">
   <property name="text">
    <string>export *.nltrack</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCAD_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wireframe Mode</string>
   </property>
  </action>
  <action name="actionShow_Track">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Track</string>
   </property>
  </action>
  <action name="actionShow_Terrain">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Terrain</string>
   </property>
  </action>
  <action name="actionShow_Sky">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Sky</string>
   </property>
  </action>
  <action name="actionShow_Ground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Ground</string>
   </property>
  </action>
  <action name="actionShow_Supports">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Supports</string>
   </property>
  </action>
  <action name="actionStart_Simulator">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Start Simulator</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionBake_for_NoLimits">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Bake for NoLimits</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
